src/screens/FeedScreen/RenderList.js

```diff
@@ -111,9 +113,10 @@ const RenderListFeed = (props) => {
     initialSetup(item);
   }, [item]);
 
+  const hasComment = getCommentLength(item.latest_reactions.comment) > 0;
+
   const contentLinkHeight = () => {
-    const haveLength =
-      getCommentLength(item.latest_reactions.comment) > 0 ? getHeightReaction() / 2.2 : 0;
+    const haveLength = hasComment ? getHeightReaction() / 2.2 : getHeightReaction() / 1.6;
     return dimen.size.FEED_CURRENT_ITEM_HEIGHT - getHeightHeader() - getHeightFooter() - haveLength;
   };
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The calculation for `contentLinkHeight` has been changed. This could potentially affect the layout of your application. Ensure that these changes do not break the UI and that they work as expected across different screen sizes and orientations.
```diff
- const haveLength =
-   getCommentLength(item.latest_reactions.comment) > 0 ? getHeightReaction() / 2.2 : 0;
- return dimen.size.FEED_CURRENT_ITEM_HEIGHT - getHeightHeader() - getHeightFooter() - haveLength;
+ const hasComment = getCommentLength(item.latest_reactions.comment) > 0;
+ const contentLinkHeight = () => {
+   const haveLength = hasComment ? getHeightReaction() / 2.2 : getHeightReaction() / 1.6;
+   return dimen.size.FEED_CURRENT_ITEM_HEIGHT - getHeightHeader() - getHeightFooter() - haveLength;
+ };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->